<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MysqlView</class>
 <widget class="QDialog" name="MysqlView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MysqlView</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>姓名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="selectButton">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="returnAllButton">
         <property name="text">
          <string>返回全表</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="commitChangeButton">
       <property name="text">
        <string>提交修改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="undoButton">
       <property name="text">
        <string>撤销修改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addRecordButton">
       <property name="text">
        <string>添加记录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delRecordButton">
       <property name="text">
        <string>删除选中行</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ascendingButton">
       <property name="text">
        <string>ID升序排列</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="descendingButton">
       <property name="text">
        <string>ID降序排列</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/mysql.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>commitChangeButton</tabstop>
  <tabstop>undoButton</tabstop>
  <tabstop>addRecordButton</tabstop>
  <tabstop>delRecordButton</tabstop>
  <tabstop>ascendingButton</tabstop>
  <tabstop>descendingButton</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>selectButton</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
